### PR TITLE
Read packages/version from env vars - 7.0.x

### PR DESCRIPTION
**Pull request by utkarsh5474**
# Description

Extension of https://github.com/confluentinc/cp-ansible/pull/918 for 7.0.x/7.1.x

Fixes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Tested as part of 918

**Test Configuration**:

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] Any variable changes have been validated to be backwards compatible

### CODE REVIEW

File: molecule/Dockerfile-debian-java8.j2 [hunk 1]
```diff
@@ -18,12 +18,17 @@ RUN apt-get update && \
 RUN /usr/bin/printf '\xfe\xed\xfe\xed\x00\x00\x00\x02\x00\x00\x00\x00\xe2\x68\x6e\x45\xfb\x43\xdf\xa4\xd9\x92\xdd\x41\xce\xb6\xb2\x1c\x63\x30\xd7\x92' > /etc/ssl/certs/java/cacerts && \
     /var/lib/dpkg/info/ca-certificates-java.postinst configure
 
-RUN wget -qO - https://packages.confluent.io/clients/deb/archive.key | sudo apt-key add - && \
-    add-apt-repository "deb https://packages.confluent.io/clients/deb/ stretch stable main" && \
+{% set PACKAGE_VER = lookup('env', 'VERSION') | default('7.0.1', true) %}
```
Comment (nsharma-git) [inline molecule/Dockerfile-debian-java8.j2:21]: I see these changes are expected in all the branches. Probably you need to update again for patches. Can we not have a simple sed/awk command to parse it from variables. 
Sorry for overlooking this earlier.

**Review by nsharma-git — CHANGES_REQUESTED** [review]

File: molecule/Dockerfile-debian-java8.j2 (hunk 1 above)
Comment (utkarsh5474) [reply molecule/Dockerfile-debian-java8.j2:21]: From which variables, @nsharma-git ?
In all jenkins jobs, I'll make an update to read these variables from molecule args
For ex: 
When someone use `molecule .. --e --e "confluent_common_repository_baseurl=xyz confluent_client_repository_baseurl=abc confluent_package_version=7.1.0"` in the jenkins jobs, a script will resolve these variables COMMON_REPO_URL, VERSION, CLIENT_REPO_URL from here.

Comment (nsharma-git) [reply molecule/Dockerfile-debian-java8.j2:21]: Can we not parse the version from https://github.com/confluentinc/cp-ansible/blob/353ddc442e4eda5b023a9f257a571a01a0fd76e4/roles/variables/defaults/main.yml#L5 in default case ?

Comment (utkarsh5474) [reply molecule/Dockerfile-debian-java8.j2]: @nsharma-git 
Please have a look. Only when this looks fine, I'll do the change for other PRs.

**Review by nsharma-git — APPROVED** [review]

**Pull request merged**